.github/workflows/e2e-tests.yml

```diff
@@ -1,14 +1,8 @@
 name: End-to-end (e2e) Tests
 
 on:
-  # Run on all pushes and on all pull requests.
-  # Prevent the "push" build from running when there are only irrelevant changes.
-  push:
-    paths-ignore:
-      - '**.md'
```

Comment:
Just in case: `paths-ignore` is [valid](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#onpushpull_requestpaths) for `pull_request` as well